src/coreclr/src/tools/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -1322,7 +1322,7 @@ protected override void AllocAndTransformManagedToNative(ILCodeStream codeStream
             ILEmitter emitter = _ilCodeStreams.Emitter;
             ILCodeLabel lNullArray = emitter.NewCodeLabel();
 
-            MethodDesc getRawSzArrayDataMethod = InteropTypes.GetRuntimeHelpers(Context).GetKnownMethod("GetRawSzArrayData", null);
```

Comment:
Does this mean we have a test hole?

Reply:
> Does this mean we have a test hole?

AFAIK proper crossgen2 testing is only getting stood up now - but I haven't been keeping track while on leave. I would expect this to be hit in one of the p/invoke interop tests with crossgen2 when large version bubble is enabled.

Cc @dotnet/crossgen-contrib

Reply:
We currently have two Crossgen2 pipelines - Pri0 and Pri1. They however haven't yet been enabled to run automatically because they are not yet 100% clean (the last time I looked I saw about a dozen failures in the Pri1 tests). I believe that we should be able to weed out the last remaining issues and add at least a limited amount of Crossgen2 testing to the pre-checkin and outerloop pipeline in 1-2 weeks.

Reply:
We only have one remaining failure in the Pri1 tests compiled with crossgen2 with GC stress stress enabled and one without (on Windows, I am in progress of fixing GC stress issues on Linux). The one failing without GC stress doesn't seem related, the one with GC stress might be related, as it fails due to an incorrect GC reporting of an array element as byref via pinvoke. But it works fine without GC stress, so maybe is it not related after all.

Reply:
Ah, syncing to the latest master, I now get failure in crossgen2 due to this.